emgena/serializers.py

```diff
@@ -104,7 +103,7 @@ def create(self, validated_data):
             ticket["Cc"] = n.cc
 
         if n.is_consent:
-            ticket["Queue"] = ena_queue
+            return 403
```

Comment:
```suggestion
            return 404
```
A 404 not found seems more appropiate.